docs/developers/index.md

```diff
@@ -58,7 +58,7 @@ Any program that runs on the Ethereum Virtual Machine (EVM) is commonly referred
 We're building a suite of language-specific landing pages for developer to learn about Ethereum in their preferred programming langauge.
 
 - [Ethereum for Java developers](/java)
-- [Ethereum for Python developers (coming soon)](#)
+- [Ethereum for Python developers](/python)
 - [Ethereum for Javascript developers (coming soon)](#)
```

Comment:
I realize this is separate from your changes but do you mind updating the "Ethereum for Javascript developers (coming soon)" & "Others coming soon!" from links to just text? I don't think presenting them as links is a good UX.